backends/timer/sdl/sdl-timer.cpp

```diff
@@ -29,11 +29,7 @@
 
 #include "common/textconsole.h"
 
-OSystem::MutexRef timerMutex;
-
```

Comment:
Is it safe to remove that mutex? I assume it was there for a reason?

Reply:
I don't think this does anything since it never gets created or deleted. There's already a similar mutex in the parent class, so it hopefully shouldn't be needed.

Reply:
In fact, I added it in the past with my (unsuccessful) attempts to fix crash at the ScummVM exit, when at some point there is a race condition with killing the timers from the audio thread.

Thus, it could be safely removed as it did not solve the problem anyway.